### PR TITLE
Apply black to the codebase

### PR DESCRIPTION
**Pull request by ericof**

### CODE REVIEW

File: .flake8 [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length = 88
```
Comment (stevepiercy) [inline .flake8]: Pylons Project projects have a [hard rule of 79-column line length](https://pylonsproject.org/community-coding-style-standards.html) under "Coding Style".

See also Pyramid's .flake8 for possibly reusable bits.
```
[flake8]
ignore =
    # E203: whitespace before ':' (black fails to be PEP8 compliant)
    E203
    # E731: do not assign a lambda expression, use a def
    E731
    # W503: line break before binary operator (flake8 is not PEP8 compliant)
    W503
    # W504: line break after binary operator (flake8 is not PEP8 compliant)
    W504
exclude =
    src/pyramid/compat.py
    tests/fixtures
    tests/pkgs
    tests/test_config/pkgs
    tests/test_config/path
show-source = True
```

File: .isort.cfg [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import_heading_stdlib=Standard Library
+import_heading_pyramid=Pyramid
+import_heading_deform=Deform
+import_heading_firstparty=Deform Demo
+lines_after_imports = 2
+not_skip=__init__.py
+include_trailing_comma=True
+force_grid_wrap=0
+combine_as_imports=True
+line_length=88
```
Comment (stevepiercy) [inline .isort.cfg]: `line_length=79`

Add carriage return at EOF.

File: deformdemo/__init__.py [hunk 3]
```diff
@@ -3,61 +3,62 @@
...
 capabilities and which provides a functional test suite  """
 
+# Standard Library
+import csv
 import decimal
 import inspect
+import logging
+import pprint
 import random
-import re
```
Comment (stevepiercy) [inline deformdemo/__init__.py]: Why were `re` and `extPat` removed?

Comment (ericof) [reply deformdemo/__init__.py]: @stevepiercy re is not used here, so it is safe to remove.
extPat was moved few lines down

File: .isort.cfg (hunk 2 above)
Comment (ericof) [reply .isort.cfg]: Fixed

**Pull request merged**